app/models/form_profile.rb

```diff
@@ -0,0 +1,110 @@
+# frozen_string_literal: true
+# TODO(AJD): Virtus POROs for now, will become ActiveRecord when the profile is persisted
+class FormFullName
+  include Virtus.model
+
+  attribute :first, String
+  attribute :middle, String
+  attribute :last, String
+  attribute :suffix, String
+end
+
+class FormAddress
+  include Virtus.model
+
+  attribute :street
+  attribute :street_2
+  attribute :city
+  attribute :state
+  attribute :country
+  attribute :postal_code
+end
+
+class FormApplicantInformation
+  include Virtus.model
+
+  attribute :full_name, FormFullName
+  attribute :date_of_birth, Date
+  attribute :gender, String
+end
+
+class FormContactInformation
+  include Virtus.model
+
+  attribute :address, FormAddress
+  attribute :home_phone, String
+end
+
+class FormProfile
+  include Virtus.model
+
+  attribute :applicant_information, FormApplicantInformation
+  attribute :contact_information, FormContactInformation
+
+  def self.mappings_for_form(form_id)
+    @mappings ||= {}
+    @mappings[form_id] || (@mappings[form_id] = load_form_mapping(form_id))
+  end
+
+  def self.load_form_mapping(form_id)
+    file = File.join(Rails.root, 'config', 'form_profile_mappings', "#{form_id}.yml")
+    raise IOError, "Form profile mapping file is missing for form id #{form_id}" unless File.exist?(file)
+    YAML.load_file(file)
+  end
+
+  # Collects data the VA has on hand for a user. The data may come from many databases/services.
+  # In case of collisions, preference is given in this order:
+  # * The form profile cache (the record for this class)
+  # * ID.me
+  # * MVI
+  # * TODO(AJD): MIS (military history)
+  #
+  def prefill_form(form_id, user)
+    @applicant_information = initialize_application_information(user)
+    @contact_information = initialize_contact_information(user)
```

Comment:
`@contact_information` is the aplicants' contact information.  To follow suit, might I suggest that `@applicant_information` be renamed to `@identity_information` since it is the applicants identity information.  Also nitpick, so feel free to ignore.